### ISSUE TITLE
[Bug] [Time] start time before current time error start time before current time error

### ISSUE THREAD

**Issue by tracywuhuan** [apache/dolphinscheduler#13447]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

when I upgrade from 3.0.x to the latest 3.0.4 version,offline scheduled task then online scheduled task, the title error will show up. But this error happened only once. After I set the start time after current time, it works and after a while offline/online operation works fine even the start time is before current time again.
<img width="1497" alt="1674960325213(1)" src="https://user-images.githubusercontent.com/6957865/215301617-d639da9f-eae9-48a7-b154-f53204519f16.png">

### What you expected to happen

no error

### How to reproduce

maybe upgrade from other version, first time to update scheduled task

### Version

3.0.x

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Radeity** [comment]
Hi, @tracywuhuan, it has been fixed in https://github.com/apache/dolphinscheduler/pull/13285.

**Comment by tracywuhuan** [comment]
thanks

**Comment by tracywuhuan** [comment]
@Radeity will it be fixed in 3.1.x?

**Comment by dengyangguang** [comment]
This is not the same issue, it is a new issue generated after fixing # 13285.

**Comment by hwfbest** [comment]
3.0的版本存在这个问题，我查阅了代码，前后端代码逻辑存在问题，前端获取开始时间时可以传当前系统时间：
![image](https://github.com/user-attachments/assets/4ab9764d-a79f-4e78-a297-b26e596f149f)
